.github/workflows/lint.yaml

```diff
@@ -0,0 +1,40 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  rustfmt:
+    name: Check Rust formatting
+    runs-on: ubuntu-latest  # OK to run from Linux since it doesn't need to build the code
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1.10.1
+        with:
+          toolchain: stable
+          components: rustfmt
+
+      - run: cargo fmt --all --check
+
+  clippy:
+    name: Check Clippy
+    runs-on: windows-latest  # required to build on Windows due to Windows-only dependencies
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1.10.1
+        with:
+          toolchain: stable
+          components: clippy
```

Comment:
This step is unnecessary and can be removed since it's already included in the base Windows runner.

Reply:
This may be slower because it won't cache. Another consideration is that we'd build on whatever toolchain the runner has instead of specifying latest stable.

Considering those two, I still think it's worth avoiding an extra dependency.